lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
+  const dows = ['Sunday','Monday','Tuesday',  'Wednesday','Thursday', "Friday",'Saturday'];
```

Comment:
There are two problems with this line.  First, it declares a data structure that is identical to one created on line 10.  Considering this is just a list of day names, it probably should only be declared once and then reused as appropriate.

Secondly, the style is all over the place.  Why is `Friday` in double quotes while everything else is single quotes?  Why are there sometimes spaces after the comma and sometimes not?  Why are there two spaces between `Tuesday` and `Wednesday`.  It's pretty clear this code doesn't conform to the project's style guide.